### DISCUSSION TITLE
Why ormar doesn't handling unknown fields in request body?

### DISCUSSION THREAD

**Discussion by staffluck** [collerek/ormar#356] [Q&A] [answered]
For example, we have a model: 
```
class Item(ormar.Model):
    class Meta:
        database = database
        metadata = metadata

    id: int = ormar.Integer(primary_key=True)
    name: str = ormar.String(max_length=100)
```
And POST endpoint to create Item. 

```
@app.post("/item/", response_model=Item)
async def create_item(item: Item):
    return await item.save()
```
If we pass an unknown field.. like "first_name" - there will be ModelError
`ormar.exceptions.ModelError: Unknown field 'first_name' for model Item`

But if we declare Pydantic model with the same fields - fastapi will handle unknown fields like there are no these fields. 

```
class ItemRequest(pydantic.BaseModel):
    class Meta:
        orm_mode = True
    name: str
    
@app.post("/item/", response_model=ItemRequest)
async def create_item(item: Item):
    return await item.save()
```
Can Ormar model handle it like the Pydantic model?

**Comment by collerek** [comment, marked as answer]
That's the same issue as described in #353, i will introduce the extra parameter on Model meta, that will allow to ignore or forbid the extra fields
